src/page/TaskPage/TaskPage.jsx

```diff
@@ -0,0 +1,72 @@
+import './TaskPage.scss'
+import { useState } from 'react';
+import { useParams } from "react-router"
+import { useNavigate } from 'react-router-dom';
+import { Header } from "../../components/Header/Header";
+import { useUserData } from "../../features/userData/useUserData.js";
+import { useTasks } from '../../features/tasks/useTasks';
+import { store } from '../../redux/store';
+
+export const TaskPage = () => {
+    const { id } = useParams();
+    const { tasks } = useTasks()
+
+    const taskToUpdate =  tasks && tasks.find(task => task.id === id)
+
+    const { userData } = useUserData()
+    const [update] = useState(id === 'create' ? false : true)
```

Comment:
1)

```suggestion
    const [update] = useState(id !== 'create')
```

2) Зачем тебе стейт который ты даже не меняешь ? Это обычная переменная